reconcile/openshift_rolebindings.py

```diff
@@ -112,7 +113,9 @@ def fetch_desired_state(ri, oc_map):
         for permission in permissions:
             cluster_info = permission["cluster"]
             cluster = cluster_info["name"]
-            namespace = permission["namespace"]
+            namespace_info = permission["namespace"]
```

Comment:
what about renaming this to `perm_namespace`?

Reply:
i did 2/3. `namespace_info` is quite common around the codebase.